Lib/tempfile.py

```diff
@@ -270,6 +270,22 @@ def _mkstemp_inner(dir, pre, suf, flags, output_type):
     raise FileExistsError(_errno.EEXIST,
                           "No usable temporary file name found")
 
+def _dont_follow_symlinks(func, path, *args):
+    # Pass follow_symlinks=False, unless not supported on this platform.
+    if func in _os.supports_follow_symlinks:
+        func(path, *args, follow_symlinks=False)
+    elif _os.name == 'nt' or not _os.path.islink(path):
```

Comment:
Why skipping the islink() test on Windows? The os.path.islink() function is available on Windows: https://docs.python.org/dev/library/os.path.html#os.path.islink

Reply:
Because on Window `os.chmod()` changes the mode of the symlink.

Reply:
I know this has been merged, but I still have two questions:

1. What happens when `func not in _os.supports_follow_symlinks`? Shouldn’t this function raise an exception or something?
2. Following Serhiy’s comment, should two conditions on line 277 be connected by `and`, i.e., don’t `chmod()` on Windows?